app/utilities/mappings.py

```diff
@@ -35,3 +35,8 @@ def _handle_sequence(value: Sequence) -> Generator[Mapping, None, None]:
                 yield from get_mappings_with_key(key, v, ignore_keys)
             if isinstance(v, Sequence):
                 yield from _handle_sequence(v)
+
+
+def get_values_for_key(key: str, data: Mapping | Sequence) -> Generator:
+    for mapping in get_mappings_with_key(key, data):
```

Comment:
Same as above.

Reply:
👍 assuming this one means for `get_mappings_with_key`? done for both anyway